solution/ui/e2e/cypress/integration/left-sidebar.spec.js

```diff
@@ -13,6 +13,17 @@ describe("Left sidebar", () => {
 
     })
 
+    it("opens the section when the left nav subsection is clicked", () => {
+        cy.viewport("macbook-15");
+        cy.visit(destination);
+        cy.get(`h3#nav-Subpart-A`).click({ force: true })
```

Comment:
it might be nice here to have an assertion that checks that the `h2#subpart-resources-heading` has the expected "Subpart A Resources" text before the section click so we ensure it changes.  That's just a personal opinion though.

Reply:
Good suggestion.